### PR TITLE
Removes IsSequencerEnclave. Updates node types.

### PR DESCRIPTION
**Pull request by joeldudley**
### Why is this change needed?

- Provide a description and a link to the underlying ticket

### What changes were made as part of this PR:

- Provide a high level list of the changes made
- List key areas for the reviewer 

### :rotating_light: Definition of done :rotating_light:
- [ ] Unit tests added to cover new or changed functionality 
- [ ] Docs pages updated to cover new or changed functionality
- [ ] [Changelog.md](https://github.com/obscuronet/go-obscuro/blob/main/docs/testnet/changelog.md) updated

### CODE REVIEW

File: tools/walletextension/main/cli.go [hunk 1]
```diff
@@ -18,7 +18,7 @@ const (
 	walletExtensionPortWSUsage   = "The port on which to serve websocket JSON RPC requests. Default: 3001."
 
 	nodeHostName    = "nodeHost"
-	nodeHostDefault = "testnet.obscu.ro"
```
Comment (otherview) [inline tools/walletextension/main/cli.go:21]: I think this was unintentional

**Review by otherview — COMMENTED** [review]
I think you might have forgotten the files in the `testnet` folder

**Comment by moraygrieve** [comment]
Do you also need to update the top level readme for the start up parameters (sequencer etc)?

https://github.com/obscuronet/go-obscuro/tree/joel/sequencer-config#building-and-running-a-local-testnet

**Comment by joeldudley** [comment]
@moraygrieve Thanks, fixed.

**Review by otherview — APPROVED** [review]

**Pull request merged**